bin/pos-cli-modules.js

```diff
@@ -6,4 +6,5 @@ program
   .name('pos-cli modules')
   .command('remove [environment] <name>', 'remove module from instance (removes configuration and data)')
   .command('list [environment]', 'list installed modules')
+  .command('pull [module] [environment]', 'pull a module')
```

Comment:
I would suggest using the same order (and argument names) as remove for consistency